server/controllers/search.js

```diff
@@ -0,0 +1,153 @@
+import db from '../models';
```

Comment:
import { Article, User, Tag } from '../models';

Reply:
this is now fixed, thank you.